### PR TITLE
RD-5340: Resolved moment and password vulnerabilities

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of updating `cloudify-ui-common` version to the newest one (an alligning installed `moment` version with it), due to the recent [vulnerabilities removal](https://github.com/cloudify-cosmo/cloudify-ui-common/pull/125)

In addition, `passport` package was updated, due to the information from `npm` that the currently used version is introducing new vulnerabilities.

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [ ] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

#### After implementing required changes

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3104/)
(failing test scenarios are mentioned on our [tracking page](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2435317761/Tracking+2022-07-01+-+2022-09-30), because of that I consider this execution a success)

#### After updating `moment` to the newest version
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3107/)

## Documentation

N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
         "better-console": "^0.2.4",
         "blueimp-md5": "^2.7.0",
         "cloudify-blueprint-topology": "^2.5.5",
-        "cloudify-ui-common": "^7.2.0",
+        "cloudify-ui-common": "^7.2.1",
         "cloudify-ui-components": "^2.17.0",
         "connected-react-router": "^6.5.2",
         "d3": "^3.5.17",
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply package.json:63]: Sure 😉

[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply package.json:63]: Package has been updated to the newest version 🎉

**Comment by NorbertAmartus** [comment]
@qooban 

> @NorbertAmartus Please also check automatic PRs created in cloudify-stage. Seems like some of them can be closed as duplicates of this PR.

Good idea 🎸 
Duplicates has been closed 🎉

**Review by qooban — APPROVED** [review]
All good 👍🏻

**Pull request merged**